### DISCUSSION TITLE
New breaking requirement of /dev/net/tun

### DISCUSSION THREAD

**Discussion by cohenchris** [qdm12/gluetun#2560] [Q&A]
Hi all, just updated my gluetun container, and my setup is now broken because of this new /dev/net/tun requirement when running the docker image. After adding it, it's fixed. I have no problem adding it, but would like to know exactly why this has been changed.

Thanks!

**Comment by mascreativelabs** [comment]
what is the new breaking requirement you're referring, can you please expand?

**Reply by cohenchris** [reply]
Searching /dev/net/tun in this repository gives you a good answer. It was recently required to add when defining your docker container, see here:

![image](https://github.com/user-attachments/assets/6b360529-fee2-48e9-92f5-7aa1a4b731e9)

Once I had updated to whichever recent version requires this, the error message in the first result is thrown.

**Comment by jmaximusix** [comment]
Thanks, had the same problem and also managed to fix it, I'm still curious why this is required now. It did work without, why is it needed now?
